Usage metrics (#305)
xataio/mdx-docs: 050-Manage/050-usage-metrics.mdx

---
title: Usage metrics

> navTitle: Usage metrics

Reviewer: ```suggestion
```

Not needed if the same.

> keywords: ['usage', 'metrics', 'limits', 'consumption']

Reviewer: ```suggestion
keywords: ['usage', 'metrics', 'limits']
```

description: Get insight into your Xata usage
slug: usage-limits
published: true
status: beta
---

> This new feature visualizes your usage of the Xata resources and features available to you at a workspace level.

Reviewer: ```suggestion
Xata tracks usage of a variety of metrics at the workspace level. Pro plan workspaces include an upgraded experience that allows for more granular monitoring of your usage over time.
```

No need to mention it as new. This doc may sit for awhile.

The following metrics are covered:

- Data storage: The storage on disk all databases in your workspace consume.
- Search storage: The storage on disk dedicated to the search and analytics engine. The search feature can be disabled in your database settings if you don't plan on using it.
- File storage: The storage on disk your file attachments consume.
- AI questions: The number of questions asked to the `ask` endpoint. AI questions are an opt-in integration with OpenAI and ChatGPT.
- Total branches: The total number of of database branches in your workspace.

> We are actively working on enhancing this feature by exposing metrics on a database branch. We will also release more metrics in time.

Reviewer: ```suggestion
For now, Xata captures metrics usage at the workspace level only.
```

Usually a good idea to avoid promises in docs.

> ### Pro usage metrics

Reviewer: ```suggestion
## Pro plan metrics
```

Accessibility. Go from h1 to h2, not h1 to h3

> For our [pro plan](/docs/concepts/pricing#pro-plan) users, Xata provides a Usage page showing detailed metrics over a number of time ranges to choose from. Additionally, we show a higher-level metrics summary for the current calendar month on the workspace landing page.

Reviewer: ```suggestion
[Pro plan](/docs/concepts/pricing) workspaces include more granular metrics over a variety of time periods.
```

I'd suggest being general here.  Reduces the need for upkeep if the defaults ever change.

![Pro workspace Usage page](images/pro-usage-details.png)

![Pro workspace overview](images/pro-usage-overview.png)

> ### Free usage metrics

Reviewer: ```suggestion
## Free plan metrics
```

> [Free plan](/docs/concepts/pricing#free-plan) users also receive a summary view on the workspace landing page. This concisely shows the percentage of available [free tier resources](/docs/concepts/pricing#pricing-plans) that have been used in the current calendar month.

Reviewer: ```suggestion
[Free plan](/docs/concepts/pricing#free-plan) workspaces highlight [usage limits](/docs/concepts/pricing) against the current calendar month.
```
